README.md

```diff
@@ -155,6 +155,7 @@ The syntax is case-insensitive.
 | Create table | `CREATE TABLE ...;` | |
 | Change table schema | `ALTER TABLE ...;` | |
 | Delete table | `DROP TABLE ...;` | |
+| Truncate table | `TRUNCATE TABLE <table>;` | Only rows are deleted. |
```

Comment:
I think users who familiar with other DBMS expects `TRUNCATE TABLE <table>` is executed atomically but it is not.
Therefore it seems better to explain about non-atomicity and implementation.
Like:
```suggestion
| Truncate table | `TRUNCATE TABLE <table>;` | Only rows are deleted. Note: Non-atomically because executed as a [partitioned DML statement](https://cloud.google.com/spanner/docs/dml-partitioned?hl=en). |
```

Reply:
Sounds good!